src/codegen/codegen_c_visitor.cpp

```diff
@@ -4121,17 +4131,17 @@ void CodegenCVisitor::print_nrn_cur_conductance_kernel(const BreakpointBlock& no
         }
         printer->add_line("double rhs = {};"_format(sum));
     }
-    if (!info.conductances.empty()) {
-        std::string sum;
-        for (const auto& conductance: info.conductances) {
-            auto var = breakpoint_current(conductance.variable);
-            sum += get_variable_name(var);
-            if (&conductance != &info.conductances.back()) {
-                sum += "+";
-            }
+
+    std::string sum;
+    for (const auto& conductance: info.conductances) {
+        auto var = breakpoint_current(conductance.variable);
+        sum += get_variable_name(var);
+        if (&conductance != &info.conductances.back()) {
+            sum += "+";
         }
-        printer->add_line("double g = {};"_format(sum));
     }
+    printer->add_line("double g = {};"_format(sum));
```

Comment:
Are we sure about this change? if conductances vector is empty, we are not supposed to emit this code double g = … ? (if there is a problem with prcellstate then you can check same condition there?)

Reply:
The function `print_nrn_cur_conductance_kernel` is called only if `conductances` is not empty.